docker-compose.ci.yml

```diff
@@ -57,9 +57,11 @@ services:
 
   sftp:
     image: atmoz/sftp:alpine
+    volumes:
+      - ./build:/home/test/upload
```

Comment:
could probably use a subfolder of `build/`, not to interact with the rest of the files there; should be automatically created by `docker-compose`

Reply:
I'll put this in another PR

Reply:
See #1931